### PR TITLE
added d0lt.mdx

### PR DESCRIPTION
**Pull request by rajshukla1102**
## Description

I have added a doc page for the D0lt data integration
**Fixes** #5155

## Type of change

- [ ] 📄 This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have updated the documentation, or created issues to update them.

### CODE REVIEW

**Comment by rajshukla1102** [comment]
`I have read the CLA Document and I hereby sign the CLA`

**Comment by rajshukla1102** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   **=** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by rajshukla1102** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by rajshukla1102** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by rajshukla1102** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by rajshukla1102** [comment]
recheck

File: docs/app-integrations/d0lt.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+* `user`: username asscociated with database
+* `password`: password to authenticate your access
+* `host`: host to server IP Address or hostname
+* `port`: port through which TCPIP connection is to be made
+* `database`: Database name to be connected
+
+
+## Usage
+In order to make use of this handler and connect to D0lt in MindsDB, the following syntax can be used,
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/app-integrations/d0lt.mdx:28]: Please use the below code block:

> \```sql
> SQL GOES HERE
> \```

File: docs/app-integrations/d0lt.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "user":"root",
+    "password":"",
+    "host":"127.0.0.1",
+    "port":3306,
+    "database":"information_schema"
+};
+~~~~
+
+Now, you can use this established connection to query your database as follows,
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/app-integrations/d0lt.mdx:42]: Please use the below code block:

> \```sql
> SQL GOES HERE
> \```

**Comment by martyna-mindsdb** [comment]
@rajshukla1102 
Please implement the above review comments in this PR.

Also, please sign our CLA by leaving a comment here that contains this line: I have read the CLA Document and I hereby sign the CLA

**Review by rajshukla1102 — COMMENTED** [review]
Hey @martyna-mindsdb can you check it once

**Comment by rajshukla1102** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by martyna-mindsdb** [comment]
@rajshukla1102 
Please remove the `d0lt.mdx` file from `docs/app-integrations/` and place it in the `docs/data-integrations/` folder.

**Comment by rajshukla1102** [comment]
@martyna-mindsdb Sorry for the trouble. I have fixed file path. Thankyou

**Comment by martyna-mindsdb** [comment]
Thanks @rajshukla1102 
No problem.

Please remove this file `docs/app-integrations/d0lt.mdx`.

**Comment by rajshukla1102** [comment]
@martyna-mindsdb done!

**Pull request merged**